grafana/greptimedb-cluster.json

```
@@ -23,7 +23,7 @@
       "type": "grafana",
       "id": "grafana",
       "name": "Grafana",
-      "version": "10.2.3"
+      "version": "11.1.3"
```

Can we keep the old grafana version to 10.x?
```suggestion
      "version": "10.2.3"
```